Lib.Web.Mvc/JQuery/JqGrid/DataAnnotations/JqGridColumnEditableAttribute.cs

```diff
@@ -167,14 +168,12 @@ public JqGridColumnEditableAttribute(bool editable)
         /// </summary>
         /// <param name="editable">If this column can be edited</param>
         /// <param name="dataUrlRouteName">Route name for the URL to get the AJAX data for the select element (if EditType is JqGridColumnEditTypes.Select) or jQuery UI Autocomplete widget (if EditType is JqGridColumnEditTypes.Autocomplete).</param>
-        public JqGridColumnEditableAttribute(bool editable, string dataUrlRouteName)
-            : this(editable)
+        public JqGridColumnEditableAttribute(bool editable, string dataUrlRouteName) : this(editable)
```

Comment:
You shouldn't be making style changes, I'm guessing you simply have differently configured Visual Studio or ReSharper but you should keep the style of the project you are contributing into.